src/promptflow/promptflow/_cli/_pf_azure/_run.py

```diff
@@ -495,15 +491,13 @@ def show_run_details(subscription_id, resource_group, workspace_name, run_name,
     pretty_print_dataframe_as_table(details)
 
 
-@exception_handler("Show run metrics")
 def show_metrics(subscription_id, resource_group, workspace_name, run_name):
     """Show run metrics from cloud."""
     pf = _get_azure_pf_client(subscription_id, resource_group, workspace_name)
     metrics = pf.runs.get_metrics(run=run_name)
     print(json.dumps(metrics, indent=4))
 
 
-@exception_handler("Stream run")
 def stream_run(subscription_id, resource_group, workspace_name, run_name, debug=False):
     """Stream run logs from cloud."""
     pf = _get_azure_pf_client(subscription_id, resource_group, workspace_name, debug=debug)
```

Comment:
After this change, if failed in `_get_azure_pf_client`, do we have red error message in CLI?

Reply:
Yes, We added these logics at the beginning of the CLI command.